### ISSUE TITLE
[Question] Party

### ISSUE THREAD

**Issue by Blunnyn** [suriyun-production/mmorpg-kit-docs#1936]
In mmo, when the "leader" of the party disconnects, should the party be eliminated?
I have noticed that even if there are 2 players in a party and they both disconnect, the party is still saved in the database.
I think that when the party leader disconnects, it should be eliminated and when it connects, create a new party.

**Comment by insthync** [comment]
No, I don't think like you.

**Comment by moepi2k** [comment]
Me neither, why should the party be eliminated? It could be that the leader just has a disconnect. Party should be persistant till someone decide to eliminate it. Thats how it works in all mmos

**Comment by LordPhrozen** [comment]
Have to echo what's been said already. The internet is a fickle thing, people lose connections to games all the time. Having to rebuild a raid party because who ever started it popped offline? would be a substantial waste of time for players.

However his second point there "I have noticed that even if there are 2 players in a party and they both disconnect, the party is still saved in the database."

**If 0 people are in a party from disconnects or otherwise, should probably kill it.**

**Comment by moepi2k** [comment]
Yes the party is still in the db because when those players login back they still in the party and can continue play together
